region/tourian/main/Metroid Room 2.json

```diff
@@ -754,7 +765,49 @@
       "note": [
         "Use a mid-air wiggle to reduce the height of Samus' hitbox after the walljump, to be able to fit through the gap without morphing."
       ],
-      "devNote": "Metroids randomly block shots fired and can make opening the door unreliable."
+      "devNote": [
+        "FIXME: Add variations for tanking the Metroid damage (taking into account how they can block the shot to open the door)"
+      ]
+    },
+    {
+      "link": [2, 1],
+      "name": "Come in Shinecharging, Leave Shinecharged (Left Side, Tricky Dash Jump)",
+      "entranceCondition": {
+        "comeInShinecharging": {
+          "length": 0,
+          "openEnd": 1
+        }
+      },
+      "requires": [
+        {"shineChargeFrames": 170},
+        "canTrickyDashJump",
+        "canShinechargeMovementTricky",
+        {"or": [
+          "f_KilledMetroidRoom2",
+          "ScrewAttack",
+          {"metroidFrames": 160}
+        ]}
+      ],
+      "exitCondition": {
+        "leaveShinecharged": {}
+      },
+      "unlocksDoors": [
+        {"types": ["super"], "requires": [
+          {"or": [
+            "f_KilledMetroidRoom2",
+            "ScrewAttack"
+          ]}
+        ]},
+        {"types": ["missiles", "powerbomb"], "requires": ["never"]}
+      ],
+      "flashSuitChecked": true,
+      "note": [
+        "Go up the left side of the room to make it out the top door with a few shinecharge frames remaining."
+      ],
+      "detailNote": [
+        "Do a big initial jump to land in the middle of the lower platform,",
```

Comment:
I preferred jumping normally and tapping run.

Reply:
Makes sense. Updated the note to include that as an option.